File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/CropYieldInformation/CropYieldInformationSummaryDto.cs
namespace OregonTilth.Models.DataTransferObjects
{
    public class CropYieldInformationSummaryDto
    {
        public int? CropYieldInformationID { get; set; }
        public int WorkbookID { get; set; }
        public CropSummaryDto Crop { get; set; }
        public CropUnitSummaryDto CropUnit { get; set; }
        public decimal HarvestedYieldPerStandardUnitOfSpace { get; set; }
        public decimal MarketableYieldPerStandardUnitOfSpace { get; set; }
        public decimal PackagingCostPerCropUnit { get; set; }
        public decimal PricePerCropUnit { get; set; }

    }
}

File: OregonTilth.Web/src/app/shared/models/forms/crop-yield-information/crop-yield-information-summary-dto.ts
import { CropDto } from "../../generated/crop-dto";
import { CropUnitDto } from "../../generated/crop-unit-dto";

export class CropYieldInformationSummaryDto {
    WorkbookID: number;
    Crop: CropDto;
    CropUnit: CropUnitDto;
    HarvestedYieldPerStandardUnitOfSpace: number;
    MarketableYieldPerStandardUnitOfSpace: number;
    PackagingCostPerCropUnit: number;
    PricePerCropUnit: number;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

